hugegraph-test/src/main/java/com/baidu/hugegraph/unit/core/RolePermissionTest.java

```diff
@@ -79,59 +85,98 @@ public void testBuiltinNone() {
         Assert.assertSame(none, RolePermission.builtin(none));
         Assert.assertSame(none, RolePermission.builtin(role1));
 
-        Assert.assertEquals("{\"roles\":{\"none\":{\"NONE\":[{\"type\":\"ALL\",\"label\":\"*\",\"properties\":null}]}}}", none.toJson());
-        RolePermission role = RolePermission.fromJson("{\"roles\":{\"none\":{\"NONE\":[{\"type\":\"ALL\",\"label\":\"write\",\"properties\":null}]}}");
+        Assert.assertEquals("{\"roles\":{\"none\":{\"NONE\":[" +
+                                    "{\"type\":\"ALL\",\"label\":\"*\",\"properties\":null}]}}}"
+                                    , none.toJson());
+        RolePermission role = RolePermission.fromJson("{\"roles\":{\"none\":{\"NONE\":[" +
```

Comment:
can you put the comma on line 90 on the previous line, and align with left parenthesis
![7](https://user-images.githubusercontent.com/13429553/167641371-55814f48-4f77-49af-9e9b-130b0f83aab8.JPG)